src/PKSim.Core/Snapshots/Classifiable.cs

```diff
@@ -0,0 +1,7 @@
+namespace PKSim.Core.Snapshots
+{
+   public class Classifiable : SnapshotBase
```

Comment:
I ended up taking the classifiables out from the nested structure in JSON. Instead, these are serialized with the name of the subject and the path of the parent classification